ui/src/core/xibo-cms.js

```diff
@@ -430,12 +430,10 @@ function XiboInitialise(scope, options) {
 
             return settings;
           },
-          filter: function(list) {
-            return $.map(list.data, function(tagObj) {
-              return {
-                tag: tagObj.type,
-              };
-            });
+          transform: function(list) {
+            return list?.data?.length > 0 ? $.map(list.data, function(tagObj) {
```

Comment:
`xibo-cms.js` isn't yet run through babel, so when we're in this file we need to make sure we don't use any JS functions which exceed our minimum browser requirements for using the CMS.

Reply:
This is noted and will updated the PR. Thanks!